addons/account/models/account_move.py

```diff
@@ -2242,7 +2242,24 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
-    def post(self):
+    def post(self, soft=False):
```

Comment:
Do we need this ?

Reply:
We don't **need** it, but since that logic should be applied every time we post automatically generated entries (never generate a posted entry in the future), I think it is better to include it here.